app/src/main/java/org/mozilla/reference/browser/EngineProvider.kt

```diff
@@ -28,6 +28,9 @@ object EngineProvider {
                 builder.crashHandler(CrashHandlerService::class.java)
             }
 
+            // Allow for exfiltrating Gecko metrics through the Glean SDK.
+            builder.telemetryDelegate(glean.GeckoAdapter());
```

Comment:
Just noticed this semi-colon at the end of the line...

Reply:
Thanks!